VideoPlayer/JAMediaReproductor/JAMediaBins.py

```diff
@@ -20,9 +20,8 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
 import gi
-gi.require_version('Gtk', '3.0')
-gi.require_version('Gst', '1.0')
 from gi.repository import Gst, GObject
+gi.require_version('Gtk', '3.0')
```

Comment:
I'm confused by this change; of the module versions originally listed, only Gst import requires a version, there is no Gtk import ... and to make it even more interesting the rest of the file doesn't use the Gst import, it uses gst, so it hasn't been tested following the port to GTK+ 3?  :grin: